webui/tests/tests/StepImplementation.ts

```diff
@@ -136,6 +142,28 @@ export default class StepImplementation {
     }
   }
 
+  /* Browser Utility Steps */
+
+  @Step('Should not have element <selector> present')
+  public async hasNoElement(selector: string) {
+    const exists = await $(selector).exists(1000);
+    assert.ok(!exists);
```

Comment:
**question:** this one I'm not sure on, but do we need an `await` here in front of `assert.ok`? 
https://docs.taiko.dev/assertions/#custom-assertions

Reply:
hmm assert.ok on its own seems synchronous https://nodejs.org/api/assert.html#assert_assert_doesnotreject_asyncfn_error_message

let me see.

Reply:
`exists` is a bool and not a promise so 150 can properly check it. i tested to see it fail if I pass in `exists` instead

Reply:
not sure why they use await in their example